src/status_im/contexts/keycard/authorise/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im.contexts.keycard.authorise.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.events-helper :as events-helper]
+            [status-im.common.standard-authentication.core :as standard-auth]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.security.core :as security]))
+
+(defn view
+  []
+  (let [profile-name        (rf/sub [:profile/name])
+        profile-picture     (rf/sub [:profile/image])
+        customization-color (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/page-nav
+      {:key        :header
+       :background :blur
+       :icon-name  :i/close
+       :on-press   events-helper/navigate-back}]
+     [quo/page-top
+      {:title       (i18n/label :t/authorise-with-password)
+       :description :context-tag
+       :context-tag {:full-name           profile-name
+                     :profile-picture     profile-picture
+                     :customization-color customization-color}}]
+     [rn/view {:style {:flex 1 :padding-horizontal 20 :justify-content :space-between}}
+      [quo/text (i18n/label :t/migrate-key-pair-authorise)]
+      [standard-auth/slide-button
+       {:size                :size-48
+        :container-style     {}
```

Comment:
not needed I guess

Reply:
it's needed , for some reason slide-button has a style with flex 1, which is strange